mne/stats/regression.py

```diff
@@ -134,3 +143,208 @@ def _fit_lm(data, design_matrix, names):
         mlog10_p_val[predictor] = -np.log10(p_val[predictor])
 
     return beta, stderr, t_val, p_val, mlog10_p_val
+
+
+def linear_regression_raw(raw, events, event_id=None,
+                          tmin=-.1, tmax=1,
```

Comment:
you could choose your line breaks in a way to minimize the number of line used. This is what we do most of the time. Especially in function definitions.